js/templates/site.es6.js

```diff
@@ -13,11 +13,11 @@ module.exports = function () {
             <li>
                 <h1 class="site-info__domain">${domain}</h1>
                 ${toggleButton(!this.model.isWhitelisted, 'js-site-toggle')}
-                <div class="site-info__rating site-info__rating--${this.model.siteRating} pull-right"></div>
+                <div class="site-info__rating site-info__rating--${this.model.siteRating}--${this.model.browser} pull-right"></div>
             </li>
             <li>
                 <h2>
-                    <span class="site-info__https-status site-info__https-status--${this.model.httpsState}"></span>
+                    <span class="site-info__https-status site-info__https-status--${this.model.httpsState}--${this.model.browser}"></span>
                     <span class="site-info__https-status-msg bold">${this.model.httpsStatusText}</span>
```

Comment:
Same here as above... the `--${this.model.browser}` would just be `-${this.model.browser}`.